mlflow/sagemaker/__init__.py

```diff
@@ -354,14 +511,148 @@ def _deploy(role, image_url, app_name, model_s3_path, run_id, region_name):
         Tags=[
             {
                 'Key': 'app_name',
-                'Value': app_name,
+                'Value': endpoint_name,
             },
         ],
     )
-    eprint("endpoint_config_arn: %s" % endpoint_config_response["EndpointConfigArn"])
+    
+    eprint("Created endpoint configuration with arn: %s" 
+            % endpoint_config_response["EndpointConfigArn"])
     endpoint_response = sage_client.create_endpoint(
-        EndpointName=app_name,
+        EndpointName=endpoint_name,
         EndpointConfigName=config_name,
         Tags=[],
     )
-    eprint("endpoint_arn: %s" % endpoint_response["EndpointArn"])
+    eprint("Created endpoint with arn: %s" % endpoint_response["EndpointArn"])
+
+def _update_sagemaker_endpoint(endpoint_name, 
```

Comment:
nit: PEP8 styling (list args in one line until 100 chars then go to next line - see other methods)

Reply:
Done.